invenio_app_rdm/records_ui/views/__init__.py

```diff
@@ -47,6 +47,11 @@ def create_blueprint(app):
         view_func=record_latest,
     )
 
+    blueprint.add_url_rule(
```

Comment:
The service ought to be defined at this moment, hence you could do simply string formatting on the route here.

Reply:
What do you mean, with what purpose? I do not follow... sorry

Reply:
It was if you needed the list of configured pid providers to put them in the URL pattern.

Now you have ``"/doi/<path:pid_value>"``, but it could be something like ``"/{schemes}/<path:pid_value>".format(schemes="|".join(pidschemes))

Let's talk IRL